std/http.ln

```diff
@@ -48,16 +48,16 @@ export fn fetch(req: Request): Result<Response> {
     connId: 0, // Unused by the client
   };
   const ro = httpreq(i);
-  if ro.isErr() {
-    return ro;
-  } else {
+  if ro.isOk() {
```

Comment:
So what's happening here is, when there's early return, it figures out the return type by checking branch-by-branch until it has a fully-realized type. In this case, since I didn't want to unwrap the error object, the variable it was returning in the error path was `Result<InternalResponse>` instead of `Result<Response>` like I wanted. Simply flipping the conditionals around fixed the inferred type here.